src/Creature.js

```diff
@@ -107,10 +109,17 @@ export default class Creature {
 
     const nearestFood = getNearestDetails(this, this.state.food, true);
 
-    if (nearestCreature && nearestCreature.ref.canEat(this) && nearestCreature.distance < this.size * 2) {
+    if (this.scareTime !== null || (nearestCreature && nearestCreature.ref.canEat(this) && nearestCreature.distance < this.size * 2)) {
       // run away
       this.color = [0, 255, 0];
-      this.runAwayFrom(nearestCreature.ref);
+      if (this.scareTime == null) {
```

Comment:
In javascript we want to use triple === @justinpaoli wheres our linting for this
But in this case we can just replace with this for easier reading
```suggestion
      if (this.scareTime) {
```

Reply:
the linting is there just checked